lib/utils.js

```diff
@@ -210,6 +211,8 @@ module.exports.processMethod = async function (msg, cfg, snapshot) {
           const requestURL = transform(msg, { customMapping: config.url });
           requestOptions.url = requestURL;
         }
+        await buildRequestBody();
```

Comment:
I recognize the downside is on the first iteration we are calling this twice, but it's ok for now.